source/brailleDisplayDrivers/alva.py

```diff
@@ -204,35 +205,48 @@ def terminate(self):
 				# We must sleep after closing the COM port, as it takes some time for the device to disconnect.
 				time.sleep(self.timeout)
 
-	def _ser6SendMessage(self, cmd, value=""):
-		if isinstance(value, (int, bool)):
-			value = chr(value)
-		self._dev.write(b"\x1b{cmd}{value}".format(cmd=cmd, value=value,))
+	def _ser6SendMessage(self, cmd: bytes, value: bytes = b"") -> None:
+		if not isinstance(value, bytes):
+			raise TypeError("Expected param 'value' to have type 'bytes'")
+		self._dev.write(b"".join(["\x1b", cmd, value]))
 
-	def _ser6OnReceive(self, data):
-		if data != b"\x1b": # Escape
+	def _ser6OnReceive(self, data: bytes):
+		"""
+		@type data: bytes

```

Comment:
Does this make any sense if the type of data is already defined in the function signature?

Reply:
Probably not, I will try to remove these if I notice them. That said, docstrings providing other useful information are still desired.